src/index.ts

```diff
@@ -15,35 +15,35 @@ export const defaults = {
   command: 'git checkout -b {branch | shellquote} && git commit --allow-empty -m {commit | shellquote}'
 };
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
-const renderer = (templates:any, name:FormatterName):parseFn => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const renderer = (templates: any, name: FormatterName): ParseFn => {
   const render = name in templates
                  ? compile(templates[name], helpers)
                  : compile(defaults[name], helpers);
 
-  /* eslint-disable @typescript-eslint/no-explicit-any */
-  return (values:any) => render({ ...fallbacks, ...values }).trim();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return (values: any) => render({ ...fallbacks, ...values }).trim();
 };
 
 interface Parser {
-  branch: parseFn,
-  command: parseFn,
-  commit: parseFn,
+  branch: ParseFn,
+  command: ParseFn,
+  commit: ParseFn,
```

Comment:
I'm slightly confused by the name here. Why "parse"? I'd have expected something like `Formatter`/`FormatFn` from the perspective of somebody using these types. 🤔

Reply:
`FormatFn` sounds great to me, I guess my naming-brain was though at some point during the extraction 🙏